noria-server/dataflow/src/lib.rs

```diff
@@ -8,6 +8,11 @@
 #![deny(unused_extern_crates)]
 #![allow(clippy::redundant_closure)]
 
+#[cfg(target_env="msvc")]
+#[global_allocator]
+static ALLOC: std::alloc::System = std::alloc::System;
```

Comment:
Do we even need to specify this? Isn't it the default?

Reply:
Actually yeah, it is not no_std build so default allocator should be enabled already.

Let me check and re-push

Reply:
Updated PR